### DISCUSSION TITLE
Edge tx no Futaba fhss in multi internal rf menu

### DISCUSSION THREAD

**Discussion by Vince3084** [EdgeTX/edgetx#5462] [Q&A] [answered]
No Futaba fhss in multi internal rf on my Radiomasters 16s tx.Running 2.8 flyingdutchman edgetx software.
Tried YouTube and forums,no help

**Comment by pfeerick** [comment]
What version of MPM firmware are you running? If you recently updated that (1.3.4.0 onwards), the MPM firmware was split into surface and air protocol firmwares, and I'm not sure if Futaba SFHSS is present in both. 

What version number is shown here? 

![image](https://github.com/user-attachments/assets/43536021-abe3-44d6-99c2-175b4312f778)

It would probably be a good idea to make sure you are on at least the latest of the 2.8.x firmwares, i.e. [v2.8.5](https://github.com/EdgeTX/edgetx/releases/tag/v2.8.5), as there were some issues with the earlier 2.8 releases and showing all the MPM protocols. (note: The only reason I'm not recommending updating to the current release (i.e. [v2.10.3](https://github.com/EdgeTX/edgetx/releases/tag/v2.10.3)) is that this is unlikely to resolve the immediate issue, and potentially add complexity due to the need to check all model settings again after updating).

**Reply by Vince3084** [reply, marked as answer]
Hi,
Thanks for info.Got in touch with Radiomasters and the issue is with European multi rf menu restricted choice.Advised to flash earlier version of software that supports more internal rf choice.
Problem sorted.

**Reply by pfeerick** [reply]
You can also flash the latest Air version of the MPM firmware, especially if you don't use any surface/car stuff... as the air version of the firmware has the most protocols.
